script/Ciel/Day 10/10_01_CIEL10_3.txt

```diff
@@ -126,7 +126,7 @@ I was a total pain in the neck all day yesterday.
 -- だからホントにもう、全面的に俺が悪かったです。これからはちゃんと先輩の忠告を聞きますから、許してください」
 // > これからはちゃんと先輩の忠告を聞きます
 // sure you will buddy
-So let me be clear, this is all 100% my fault.
+So let me be clear, this was all 100% my fault.
```

Comment:
Might keep old ver just because it's still his fault / situation is still ongoing